index.js

```diff
@@ -21,13 +21,8 @@ const childProcess = execa(
   // @ts-ignore args[0] cannot be undefined here due to the prior check
   args.shift(),
   args,
+  { stdio: ['inherit', 'inherit', 'inherit'] },
```

Comment:
Do you know if there is any difference between an array and 

```suggestion
  { stdio: "inherit" },
```

We can switch over to the latter if there are none.